frontend/test/pytest/test_jit_behaviour.py

```diff
@@ -574,30 +610,12 @@ def addi(x: int, y: int):
 
 
 class TestArraysInHamiltonian:
-    @pytest.mark.parametrize(
-        "coeffs",
-        [
-            (np.array([0.4, 0.7])),
-            ([0.4, 0.7]),
-            (jnp.array([0.4, 0.7])),
-        ],
-    )
-    def test_array_repr_from_context(self, coeffs, backend):
-        """Test array representation from context in Hamiltonian."""
-
-        @qjit(target="mlir")
-        @qml.qnode(qml.device(backend, wires=6))
-        def f():
-            qml.Hadamard(wires=0)
-            qml.CNOT(wires=[0, 1])
-            obs = [qml.PauliX(0) @ qml.PauliZ(1), qml.Hadamard(0)]
-            return qml.expval(qml.Hamiltonian(coeffs, obs))
```

Comment:
Why did we remove this one?

Reply:
Oops! Fixed [9d6d2b1](https://github.com/PennyLaneAI/catalyst/pull/221/commits/9d6d2b191949b0d84f66defc7cef70f60d83f172)